specification/liftrqumulo/Qumulo.Storage.Management/tspconfig.yaml

```diff
@@ -24,6 +24,13 @@ options:
     flavor: "azure"
     packageDetails:
       name: "@azure/arm-qumulo"
+  "@azure-tools/typespec-csharp":
+    emitter-output-dir: "{project-root}/azure-sdk-for-net/sdk/qumulo/{namespace}/src"
```

Comment:
We will not specify the `emitter-output-dir` here. We will calculate from `service-dir` and `package-dir`